content/docs/user-guide/index.md

```diff
@@ -1,51 +1,72 @@
-# Overview
+# What is DVC?
+
+DVC is a [free], open-source tool that standardizes machine learning workflows.
+It helps you [manage data] and ML models, define reproducible [ML pipelines],
+and [manage experiments][experiment management], all under the same set of
+principles:
```

Comment:
Questions:

- Is this accurate and clear?
- Is the terminology good?
- Difference vs. info in https://dvc.org/doc ?
- " in home page (under redesign) ?